spec/spec.md

```diff
@@ -24,11 +24,11 @@ Presentation Exchange
 
 ## Abstract
 
-A common activity between peers in identity systems that feature the ability to generate self-asserted and third-party issued claims is the demand and submission of proofs from a Subject to a Verifier. This flow implicitly requires the Subject and Verifier have a known mechanism to facilitate the two primary steps in a proofing exchange: the way Verifiers define the proof requirements, and how Subjects must encode submissions of proof to align with those requirements.
+A common activity between peers in identity systems that feature the ability to generate self-asserted and third-party issued claims is the demand and submission of proofs from a Prover (Holder) to a Verifier. This flow implicitly requires the Holder and Verifier have a known mechanism to facilitate the two primary steps in a proving exchange: the way Verifiers define the proof requirements, and how Provers must encode submissions of proof to align with those requirements.
```

Comment:
is prover the new terminology? I see no references of it in the VC Data Model. Just subject - https://w3c.github.io/vc-data-model/#credential-subject

Reply:
"prover" is the term used to describe the entity that makes a VP... its a synonym for holder all the time I think... just makes it a little clearer "what" they are doing.